staging/traefik/templates/tests/test-configmap.yaml

```diff
@@ -12,4 +13,4 @@ data:
     @test "Test Access" {
       curl -D - http://{{ template "traefik.fullname" . }}/
     }
-
+{{end}}
```

Comment:
This doesn't need to be `{{- end}}`? You started the `{{- if` that way

Reply:
thx good catch ;)